Sources/CodableTransform.swift

```diff
@@ -33,8 +33,13 @@ open class CodableTransform<T: Codable>: TransformType {
 
     public typealias Object = T
     public typealias JSON = Any
-
-    public init() {}
+    let encoder: JSONEncoder
+    let decoder: JSONDecoder
```

Comment:
Maybe you should consider making those properties private. What do you think?

Reply:
YES, serializers should be private.